bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -122,70 +128,128 @@ async def autoreview_loop(self) -> None:
             log.info("Running check for users to nominate.")
             await self.reviewer.maybe_review_user()
 
-    @nomination_group.command(
-        name="nominees",
-        aliases=("nominated", "all", "list", "watched"),
-        root_aliases=("nominees",)
```

Comment:
`!tp list` is still one less character ;) I can add it back if you want it but i'd prefer keeping it removed.

FWIW I removed `all` and `watched` because I feel like it's harder to learn the cog commands if see them used with different invocations.